app/app/tests.py

```diff
@@ -0,0 +1,18 @@
+import pytest
```

Comment:
Why does this file exist twice? there should be no tests outside of the /tests directory.

Reply:
Done wait for commit

Reply:
Addressed in 
0785a71